dingo/pipe/importance_sampling.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, args, unknown_args):
 
         # Samples to run on
         self.proposal_samples_file = args.proposal_samples_file
-        self.event_data_file = args.event_data_file
+        self.event_data_file = args.event_data_files[0]
```

Comment:
Again, `event_data_files` looks like `file1.hdf5 file2.hdf5 ...` for me. Here we operate on a single file, so 

```
self.event_data_file = args.event_data_files[0].split()[0]
```

should fix it.

Reply:
I pushed a commit to fix this and the similar issue above.